### ISSUE TITLE
Protobuf should be updated to 3.6.1

### ISSUE THREAD

**Issue by perezd** [google/bazel-common#47]
Would be good to update the version of protobuf in here to the latest:
https://github.com/protocolbuffers/protobuf/releases/tag/v3.6.1

Currently getting compilation errors related to a very old deprecation being fully removed:

```
The value 'REPOSITORY_NAME' has been removed in favor of 'repository_name()', please use the latter (https://docs.bazel.build/versions/master/skylark/lib/native.html#repository_name). You can temporarily allow the old name by using --incompatible_package_name_is_a_function=false
```

Bazel version:
```
Build label: 0.21.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Dec 19 12:58:44 2018 (1545224324)
Build timestamp: 1545224324
Build timestamp as int: 1545224324
```

**Comment by rocketraman** [comment]
> Also, that'd get rid of:
> ...
> Context: protocolbuffers/protobuf#3781

Nope, it won't: that fix isn't in 3.6.1.

**Comment by perezd** [comment]
Oh right, that is just in master, they haven't cut that release yet.
